packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -11,10 +11,12 @@ export interface Props extends Omit<HTMLProps<HTMLInputElement>, 'value'> {
   value?: boolean;
   /** Make switch's background and border transparent */
   transparent?: boolean;
```

Comment:
Completely unrelated to your PR... This only applies to `InlineSwitch`, why is it a prop of `Switch` as well????????? Just venting.
Do you mind at least changing the comment to say this applies only to InlineSwitch?

Reply:
Yes, @JoaoSilvaGrafana, I don't mind to add it